src/ert/scheduler/lsf_driver.py

```diff
@@ -524,3 +525,30 @@ def _build_resource_requirement_arg(self) -> List[str]:
 
     async def finish(self) -> None:
         pass
+
+    def read_stdout_and_stderr_files(
+        self, runpath: str, job_name: str, num_characters_to_read_from_end: int = 300
+    ) -> str:
+        error_msg = ""
+        stderr_file = Path(runpath) / (job_name + ".LSF-stderr")
+        if msg := get_info_from_text_outfile(
+            stderr_file, num_characters_to_read_from_end
+        ):
+            error_msg += f"\n\t LSF-err: {msg}"
```

Comment:
tab characters...

Reply:
These I kept on purpose but can remove them :+1:

Reply:
Use 4 or 8 explicit spaces if you want to do formatting. Will lineshift and indents always look good? It might make it more difficult to parse the logs, but it will look good to humans.